components/CMakeLists.txt

```
@@ -8,3 +8,7 @@ if(CONFIG_AMBIQ_COMPONENT_USE_BT)
     add_subdirectory(bluetooth)
     zephyr_include_directories(bluetooth)
 endif()
+
+if(CONFIG_NEMA_AMBIQ)
```

Should follow our naming conventions
```suggestion
if(CONFIG_AMBIQ_COMPONENT_USE_NEMASDK)
```